source/stylesheets/_tab-code_signing.scss.erb

```diff
@@ -15,106 +15,131 @@
 
 	section.certificate-help {
 		position: relative;
-		@include margin-y(30px);
-		padding: 25px $section-horizontalspacing;
+		margin: 25px $section-horizontalspacing;
+		padding: 12px 18px;
+		border: 1px solid #3bc3a3;
+		@include border-radius($def-radius);
 		background-color: $def-green-ultralight;
 
-		ul {
-			@include margin-y(0);
-			padding-left: 0;
-			list-style: none;
+		.toggleabble-support-info {
+			@include display-flex;
+			@include justify-content(space-between);
+			@include align-items(flex-start);
+
+			&.open {
+
+				ul + button.arrow svg {
```

Comment:
Not wrong, but the `ul +` part is not really needed: we are not targetting specifically those arrow buttons which come after a `ul`, instead we are simply targetting arrow buttons.